src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -37,7 +37,10 @@ import SwapHeader from 'components/swap/SwapHeader'
 import TokenWarningModal from 'components/TokenWarningModal'
 import { TOKEN_SHORTHANDS } from 'constants/tokens'
 import { useAllTokens, useCurrency } from 'hooks/Tokens'
-import { ApprovalState /*, useApprovalOptimizedTrade*/, useApproveCallbackFromTrade } from 'hooks/useApproveCallback'
+import {
+  ApprovalState /*, useApprovalOptimizedTrade, OptionalApproveCallbackParams,*/,
```

Comment:
so this comes from the way we modify Uniswap files. Because we want to keep the original code the same, we comment out stuff we don't use.

That way when we do backmerges we can see what was changed in the diff

Makes it very awkward and messy but it does simplify the merge a ton